FlowCrypt/Functionality/Migration/DBMigrationService.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  DBMigrationService.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 19.02.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+import Promises
+import RealmSwift
+
+protocol DBMigration {
+    func performMigrationIfNeeded() -> Promise<Void>
+}
+
+struct DBMigrationService {
```

Comment:
We could probably remove all previous migrations - after we release 0.2.0 (it's been a while, and not many users since it was in beta and then expired for a while). I'll make another issue for later.

Reply:
great point 👍